spec/attribute.dd

```diff
@@ -817,4 +810,5 @@ $(SPEC_SUBNAV_PREV_NEXT property, Properties, pragma, Pragmas)
 )
 
 Macros:
+        CHAPTER=8
         TITLE=Attributes
```

Comment:
Thanks! I guess it's about the same amount of work because the assignment of chapter numbers is arbitrary. Putting them all in the script has the nice effect of keeping the mapping in a single place. I'll keep it in mind.